doc/syntax.html

```diff
@@ -1333,8 +1333,9 @@ <h3>List</h3>
 <section id="code-block" class="level3">
 <h3>Code block</h3>
 <p>A code block starts with a line of three or more consecutive backticks,
-optionally followed by whitespace and a language specifier, but nothing
-else. It ends with a line of backticks equal or greater in length to the
+optionally followed by a language specifier, but nothing else. The language
+specifier may optionally be preceded and/or followed by whitespace.
+It ends with a line of backticks equal or greater in length to the
```

Comment:
Suggest "The code block" instead of "It", which might be thought to refer to "the language specifier."
Also maybe put the sentence "The language specifier may..." in parentheses.

Reply:
Done.